src/js/pages/common.ts

```diff
@@ -0,0 +1,46 @@
+import render from '../utils/render';
+import { include } from '../libs/index';
+import { $, $closest } from '../utils/dom';
+import { Message, StorageKey } from '../types/index';
+import { getData, removeData } from '../utils/storage';
+
+export const closeModal = (): void => {
+  $('.modal').classList.remove('open');
+};
+
+export const validateValue = <Value>(
+  value: Value,
+  predicateList: ((value: Value) => boolean)[]
+): boolean => {
+  for (let i = 0; i < predicateList.length; i++) {
+    if (!predicateList[i](value)) return false;
+  }
+  return true;
```

Comment:
Array의 find method를 사용해보면 어떨까요?
```suggestion
  return !predicateList.find(v => v(value));
```
혹은
```suggestion
  return !find(predicateList, v => v(value));
```
이런식으로 표현할 수 있을 것 같습니다.

Reply:
메서드 사용을 최대한 줄여보고자 find 함수를 만들어놓고 사용을 안하고 있었네요;;;